app/controllers/coronavirus/reorder_announcements_controller.rb

```diff
@@ -26,10 +26,10 @@ def update
       end
 
       if success
-        message = "Announcements were successfully reordered."
+        message = helpers.t("coronavirus.reorder_announcements.update.success")
```

Comment:
Just curious (non blocker) but why does this need `helpers.t` and the other ones just need `t`?

Reply:
It's because it's in the controller. `t` is a helper method that's only available in views. The other way to write this is `I18n.t`, but using `helpers` allows the controller to access the same helper method that the view uses.